resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json

```
@@ -1,13 +1,225 @@
+      "maximum": 99999999999.99,
+      "example": 160
+    },
+    "netProfit": {
+      "title": "netProfit",
+      "description": "The net profit for the business. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+      "type": "number",
+      "multipleOf": 0.01,
+      "minimum": 0,
+      "maximum": 99999999999.99,
+      "example": 160
+    },
+    "netLoss": {
+      "title": "netLoss",
+      "description": "The net loss for the business. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+      "type": "number",
+      "multipleOf": 0.01,
+      "minimum": 0,
+      "maximum": 99999999999.99,
+      "example": 160
+    },
+    "totalAdditions": {
+      "title": "totalAdditions",
+      "description": "The total additions to the net profit (or deductions from the net loss) for the business. This value must be between -99999999999.99 and 99999999999.99 up to 2 decimal places.",
+      "type": "number",
+      "multipleOf": 0.01,
+      "minimum": -99999999999.99,
+      "maximum": 99999999999.99,
+      "example": 160
+    },
+    "totalDeductions": {
+      "title": "totalDeductions",
+      "description": "The total deductions from the net loss (or additions to the net profit) for the business. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+      "type": "number",
+      "multipleOf": 0.01,
+      "minimum": 0,
+      "maximum": 99999999999.99,
+      "example": 160
+    },
+    "accountingAdjustments": {
+      "title": "accountingAdjustments",
+      "description": "The taxable profit after all additions and deductions have been applied for the business. This value must be between -99999999999.99 and 99999999999.99 up to 2 decimal places.",
+      "type": "number",
+      "multipleOf": 0.01,
+      "minimum": -99999999999.99,
+      "maximum": 99999999999.99,
+      "example": 160
+    },
+    "taxableProfit": {
+      "title": "taxableProfit",
+      "description": "The adjusted Income Tax loss after all additions and deductions have been applied for the business. This value must be between 0 and 99999999999. No decimals.",
+      "type": "number",
+      "multipleOf": 1,
+      "minimum": 0,
+      "maximum": 99999999999,
+      "example": 160
+    },
+    "adjustedIncomeTaxLoss": {
+      "title": "adjustedIncomeTaxLoss",
+      "description": "The adjusted Income Tax loss after all additions and deductions have been applied for the business. This value must be between 0 and 99999999999. No decimals.",
+      "type": "number",
+      "multipleOf": 1,
+      "minimum": 0,
+      "maximum": 99999999999,
+      "example": 160
+    },
+    "totalBroughtForwardIncomeTaxLosses": {
+      "title": "totalBroughtForwardIncomeTaxLosses",
+      "description": "The total sum of Income Tax losses brought forward from earlier years for the business. This value must be between 0 and 99999999999. No decimals.",
+      "type": "number",
```

For clarity those that don't support decimals should probably be declared as type "integer" for example:
```suggestion
      "type": "integer",
```